src/Symfony/Component/Form/Test/TypeTestCase.php

```diff
@@ -34,6 +36,20 @@ protected function setUp()
         $this->builder = new FormBuilder(null, null, $this->dispatcher, $this->factory);
     }
 
+    protected function getTypedExtensions()
```

Comment:
If you want this to be called automatically based on the parent `FormIntegrationTestCase` class, you will have to rename this method to `getTypeExtensions()`.